cmd/collectors/rest/plugins/shelf/shelf.go

```diff
@@ -296,5 +306,154 @@ func (my *Shelf) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 		return true
 	})
 
+	return my.calculateEnvironmentMetrics(output, data)
+}
+
+func (my *Shelf) calculateEnvironmentMetrics(output []*matrix.Matrix, data *matrix.Matrix) ([]*matrix.Matrix, error) {
```

Comment:
output var hasn't been used inside function, right ?

Reply:
not needed. corrected.